ansible/roles/ironic/templates/ironic-inspector.conf.j2

```diff
@@ -93,3 +93,8 @@ backend_url = {{ redis_connection_string }}
 # tooz defaults to a newer version, we should explicitly specify `v3`
 backend_url = etcd3+{{ internal_protocol }}://{{ kolla_internal_fqdn }}:{{ etcd_client_port }}?api_version=v3{% if openstack_cacert %}?ca_cert={{ openstack_cacert }}{% endif %}
 {% endif %}
+
+{% if ironic_inspector_known_devices_file_path is defined %}
+[accelerators]
+known_devices = /etc/ironic-inspector/known_devices.yaml
+{% endif %}
```

Comment:
I suspect that this change does not relate to the IPsec and comes from upstream, 
Be sure that we clean this branch (rebase) on the latest upstream before we push it upstream.